cmd/edit/machinepool/nodepool.go

```diff
@@ -24,11 +24,12 @@ func editNodePool(cmd *cobra.Command, nodePoolID string, clusterKey string, clus
 	isAutoscalingSet := cmd.Flags().Changed("enable-autoscaling")
 	isLabelsSet := cmd.Flags().Changed("labels")
 	isTaintsSet := cmd.Flags().Changed("taints")
-	isLabelOrTaintSet := isLabelsSet || isTaintsSet
 	isVersionSet := cmd.Flags().Changed("version")
 	isAutorepairSet := cmd.Flags().Changed("autorepair")
 	isTuningsConfigSet := cmd.Flags().Changed("tuning-configs")
 
+	isAnyAdditionalParameterSet := isLabelsSet || isTaintsSet || isTuningsConfigSet
```

Comment:
Perhaps worth to extend this to version, and autorepair?

Would you mind checking if that works too (it was somehow on my list but I think is a similar code like this) ?

Thanks

Reply:
added and validated version + autorepair behaviour